scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigquery/avro/ToTableRow.scala

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2019 Spotify AB.
```

Comment:
should this be 2022?

Reply:
\+ add the copyright header to other new classes?

Reply:
I'm copying old code, so I don't think so? I have a vague memory that it's not actually required to have the date in the headers anyway.  Will add to other files.